app/src/main/java/app/model/service/Service.java

```diff
@@ -31,20 +38,31 @@ public class Service {
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
+    @Column(unique = true)
```

Comment:
Should this be removed? Two or more jurisdictions can have the same Service Code.

Reply:
you are right. It stays in after merging with the main branch